src/code/z_skin_awb.c

```diff
@@ -1,13 +1,203 @@
 #include "global.h"
+#include "z64skin.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin_awb/func_80138410.s")
+void func_80138410(Skin* skin) {
+    skin->skeletonHeader = NULL;
+    skin->limbCount = 0;
+    skin->vtxTable = NULL;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin_awb/func_80138424.s")
+/**
+ * Initialises the Vtx buffers used for limb at index `limbIndex`
+ */
+void Skin_InitAnimatedLimb(GameState* gameState, Skin* skin, s32 limbIndex) {
+    s32 i;
+    SkinLimb** skeleton = Lib_SegmentedToVirtual(skin->skeletonHeader->segment);
+    SkinAnimatedLimbData* animatedLimbData =
+        Lib_SegmentedToVirtual(((SkinLimb*)Lib_SegmentedToVirtual(skeleton[limbIndex]))->segment);
+    SkinLimbModif* temp_v0 = Lib_SegmentedToVirtual(animatedLimbData->limbModifications);
```

Comment:
```suggestion
    SkinLimbModif* modifications = Lib_SegmentedToVirtual(animatedLimbData->limbModifications);
```
Also, `limbModifications` is a case of using the full word `modifications` instead of `motif`, so it's currently inconsistent